gh-106318: Add example for str.isdecimal() (#137559)
python/cpython: Doc/library/stdtypes.rst

   otherwise. Decimal characters are those that can be used to form

>    numbers in base 10, e.g. U+0660, ARABIC-INDIC DIGIT

Reviewer: ```suggestion
   numbers in base 10, such as U+0660, ARABIC-INDIC DIGIT
```

   ZERO.  Formally a decimal character is a character in the Unicode
   General Category "Nd". For example:

   .. doctest::

      >>> '0123456789'.isdecimal()
      True

>       >>> '٠١٢٣٤٥٦٧٨٩'.isdecimal() # ARABIC-INDIC DIGIT ZERO TO NINE

Reviewer: ```suggestion
      >>> '٠١٢٣٤٥٦٧٨٩'.isdecimal()  # Arabic-Indic digits zero to nine
```

   like the Kharosthi numbers.  Formally, a digit is a character that has the
   property value Numeric_Type=Digit or Numeric_Type=Decimal.

> 

Reviewer 2: This change is wrong, the other have a 2 line gap.

Author: Sorry. Isn't a linter/pre-commit for this?

Reviewer 2: We do not have a pre-commit check for such things unfortunately.

Author: Done.
